packages/neuron-ui/src/components/SpecialAssetList/hooks.ts

```diff
@@ -15,9 +14,33 @@ import {
   useDialogWrapper,
   useDidMount,
 } from 'utils'
+import { TFunction } from 'i18next'
 import { MILLISECONDS, MILLISECONDS_PER_DAY } from 'utils/const'
 import { AssetInfo, ChequeAssetInfo, NFTType } from '.'
 
+interface SpecialAssetCell {
```

Comment:
I see there is also a definition of this type in `SpecialAssetList/index.tsx`.
Should we consider having `index.tsx` import this type from `hooks.ts`?